lua/autorun/server/sv_downloader_init.lua

```diff
@@ -24,13 +27,46 @@ local context = {
     usingAddons = {},
     started = SysTime(),
     gamemodeAddons = {},
-    addonsToCache = {}
+    addonsToCache = {},
+    manualAddons = {},
+    scanResult = {}
 }
 
 if not file.Exists(context.dataFolder, "DATA") then
     file.CreateDir(context.dataFolder)
 end
 
+net.Receive("uwd_exchange_scan_result", function(len, ply)
+    local scanResult = table.Copy(context.scanResult)
```

Comment:
Using the context variable inside this scope will prevent GC to collect it, on dedicated servers people will rarely use this

Reply:
To fix this GC issue I created a new module to handle the GUI data with its own subcontext
05deba41dea61d55bb0afb47237100c3aca009f1

Reply:
I'd like you to take a look into this commit before accepting the request as resolved.